doc/rm/c_cpp_coding_style.md

```diff
@@ -143,6 +143,19 @@ The [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html)
 Consequently, C++-specific rules don't apply.
 In addition to the shared C and C++ style guide rules outlined before, the following C-specific rules apply.
 
+### C Version
+
+***C code should target C11.***
+
+The following nonstandard extensions may be used:
+*   Inline assembly
+*   Nonstandard Attributes
```

Comment:
Nit: for this and the next line, Attributes and Builtins surely don't need to be capitalised? Or alternatively, capitalise "assembly" in the line above for consistency.

Reply:
Updated